src/main/java/org/sagebionetworks/web/client/widget/entity/act/ApproveUserAccessModal.java

```diff
@@ -19,6 +19,7 @@
 import org.sagebionetworks.repo.model.table.Row;
```

Comment:
Made a bunch of IDE-suggested "clean up" changes. Only logical change is pointed out below.